themes/bootstrap3/js/lightbox.js

```diff
@@ -154,7 +139,7 @@ VuFind.register('lightbox', function Lightbox() {
         }
         if ( // Close the lightbox after deliberate login
           obj.method                                                                 // is a form
-          && ((obj.url.match(/MyResearch/) && !obj.url.match(/Bulk/))                // that matches login/create account
+          && ((obj.url.match(/MyResearch/) && !obj.url.match(/Bulk/) && !obj.url.match(/Delete/)) // that matches login/create account
```

Comment:
No, those are still needed for the login event to fire, and if you'd like to get rid of the automatic refresh on login, any login form displayed in the lightbox would need to have data-lightbox-close="VuFind.refreshPage();" or something.